website/docs/usage/index.mdx

```diff
@@ -259,6 +259,26 @@ source code and recompiling frequently.
   $ python setup.py develop
   ```
 
+#### Additional tooling
```

Comment:
I would make the headline here more explicit and not have another sub-headline for quickstart.

```suggestion
#### Visual Studio Code extension
```